docs/pages/2019_MARVEL_Psik_MaX/sections/setup.rst

```diff
@@ -161,9 +161,8 @@ Getting help
 There are a number of helpful resources available to you for getting more information about AiiDA.
 Please consider:
 
- * consulting the extensive `AiiDA documentation <https://aiida-core.readthedocs.io/en/stable/>`_
+ * consulting the extensive `AiiDA documentation <https://aiida-core.readthedocs.io/en/v1.0.0b1/>`_
```

Comment:
```suggestion
 * consulting the extensive `AiiDA documentation <https://aiida-core.readthedocs.io/en/latest/>`_
```
I get the point to use a fixed version here but for the moment let's put `latest` since the aiida-core release for the tutorial isn't out yet.